From bb38e6813356ad152b0b3e9ae7da4bff69d50002 Mon Sep 17 00:00:00 2001
From: Jake McLaughlin <jake.mclaughlin98@gmail.com>
Date: Wed, 27 Sep 2023 18:23:35 -0400
Subject: [PATCH 2/2] OpenMP patch

---
 fuse_optimizers/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/fuse_optimizers/CMakeLists.txt b/fuse_optimizers/CMakeLists.txt
index 7987873..d1b7528 100644
--- a/fuse_optimizers/CMakeLists.txt
+++ b/fuse_optimizers/CMakeLists.txt
@@ -12,6 +12,8 @@ set(build_depends
   std_srvs
 )
 
+find_package(OpenMP)
+
 find_package(catkin REQUIRED COMPONENTS
   ${build_depends}
 )
@@ -47,12 +49,16 @@ target_include_directories(${PROJECT_NAME}
 )
 target_link_libraries(${PROJECT_NAME}
   ${catkin_LIBRARIES}
+  ${OpenMP_CXX_LIBRARIES}
 )
 set_target_properties(${PROJECT_NAME}
   PROPERTIES
     CXX_STANDARD 14
     CXX_STANDARD_REQUIRED YES
 )
+target_compile_options(${PROJECT_NAME}
+  PRIVATE ${OpenMP_CXX_FLAGS}
+)
 
 ## batch_optimizer node
 add_executable(batch_optimizer_node
-- 
2.25.1

